.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: scala
 scala:
-  - 2.10.4
+  - 2.12.4
```

Comment:
You may want to upgrade to 2.12.7 here, see https://www.scala-lang.org/news/2.12.7